packages/shared-ux/card/no_data/src/no_data_card.tsx

```diff
@@ -7,37 +7,56 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import React, { FunctionComponent } from 'react';
-import { EuiButton, EuiCard, EuiScreenReaderOnly } from '@elastic/eui';
+import React, { MouseEventHandler, ReactNode } from 'react';
+import { EuiButton, EuiCard, EuiScreenReaderOnly, EuiCardProps } from '@elastic/eui';
 
-import type { NoDataCardProps } from './types';
 import { NoDataCardStyles } from './no_data_card.styles';
 
-const defaultDescription = i18n.translate(
-  'sharedUXComponents.pageTemplate.noDataCard.description',
-  {
-    defaultMessage: `Proceed without collecting data`,
-  }
-);
+const defaultDescription = i18n.translate('sharedUXPackages.card.noData.description', {
+  defaultMessage: `Proceed without collecting data`,
+});
 
-export const NoDataCard: FunctionComponent<NoDataCardProps> = ({
+/**
+ * Props for the `NoDataCard` component.
+ */
+export type Props = Partial<Omit<EuiCardProps, 'layout'>> & {
+  /**
+   * Provide just a string for the button's label, or a whole component;
+   * The button will be hidden completely if `isDisabled=true`
+   */
+  button?: string | ReactNode;
+  /** Remapping `onClick` to any element */
```

Comment:
What does this mean?

Reply:
The comment was there before, so not sure... I'll revise.